release-notes.md

```diff
@@ -3,6 +3,43 @@
 See [How to upgrade Anbox Cloud](https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750) or [How to upgrade the Anbox Cloud Appliance](https://discourse.ubuntu.com/t/upgrade-anbox-cloud-appliance/24186) for
 instructions on how to update your Anbox Cloud deployment.
 
+
+[Details=1.17.0]
+
+# 1.17.0 (February 15 2023)
+
+#### New features & improvements
+
+* Enabling work of Vulkan support for NVIDIA GPUs
+* Dynamic screen resolution support
+* Start and stop for containers in AMS
+* Various dashboard improvements
+* Experimental support for software accelerated AV1 encoding is now available behind a [feature flag](https://anbox-cloud.io/docs/ref/ams-configuration) (`experimental.force_av1_software_encoding`).
```

Comment:
Shouldn't this be a discourse link?

Reply:
Yes, in the release notes we should link to Discourse. In the announcement we link to the doc output.